lib/p2p/Pool.ts

```diff
@@ -166,14 +167,14 @@ class Pool extends EventEmitter {
    * @param ignoreKnown whether to ignore nodes we are already aware of, defaults to false
    * @returns a promise that will resolve when all outbound connections resolve
    */
-  private connectNodes = (nodes: NodeConnectionIterator, ignoreKnown = false) => {
+  private connectNodes = (nodes: NodeConnectionIterator, ignoreKnown?: boolean, retryConnecting?: boolean) => {
```

Comment:
I know it functions the same for purposes of false checks, but this technically contradicts the comments above that says `ignoreKnown` defaults to false. Personally I think it's a bit more clear too to default boolean parameters to `false` rather than `undefined`, unless there's a reason to do it this way that I'm not aware of.